.buildkite/scripts/run_models.sh

```diff
@@ -17,5 +17,5 @@ echo `pwd`
 cd integration_tests
 dbt deps
 dbt seed --target "$db" --full-refresh
-dbt run --target "$db" --full-refresh --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false, ad_reporting__facebook_ads_enabled: false}' 
-dbt test --target "$db" --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false, ad_reporting__facebook_ads_enabled: false}' 
+dbt run --target "$db" --full-refresh --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false, ad_reporting__facebook_ads_enabled: false, ad_reporting__reddit_ads_enabled: false}' 
```

Comment:
Since this is the first version to support Reddit Ads, we should test it in our buildkite job.

I think we can move maybe Amazon Ads to be disabled for now, but we should test Reddit Ads this time around.

Reply:
oh duh yes...  removed reddit, added amazon